### PR TITLE
Update main.c

### PR DESCRIPTION
**Pull request by QiangChen001**

### CODE REVIEW

**Comment by QiangChen001** [comment]
inserted the mergesort

File: main.c [hunk 1]
```diff
@@ -15,9 +15,51 @@ const char *SORT_NAMES[] = {"Merge Sort", "Quick Sort", "Bucket Sort"};
...
 
-// }
+    if (start < end){
+
+      int mid = start+(end-start)/2;
+
+      int pointer_left = start;
+      int pointer_right = mid+1;
+
+      inversion_MergeSort += MergeSortAndCount(a, pointer_left, mid);
```
Comment (quintenBruce) [inline main.c]: MergeSortAndCount(...) is not recognized. Perhaps it is incorrectly named or the function is missing or out of scope?

File: main.c [hunk 2]
```diff
@@ -15,9 +15,51 @@ const char *SORT_NAMES[] = {"Merge Sort", "Quick Sort", "Bucket Sort"};
...
+      inversion_MergeSort += MergeSortAndCount(a, pointer_right, end);
```
Comment (quintenBruce) [inline main.c]: The identifier 'a' is undeclared. Perhaps you forgot to declare this variable?

**Review by quintenBruce — CHANGES_REQUESTED** [review]
I'm getting a few errors when I run 'main.c'. 

I added comments where an error is occurring.

File: main.c (hunk 2 above)
Comment (QiangChen001) [reply main.c]: oh, we use different name of the array variable "a" and"arr".

File: main.c (hunk 1 above)
Comment (QiangChen001) [reply main.c]: Yes, we use different name for the function, and I forgot to change it in the recursions when inserted.

**Review by quintenBruce — APPROVED** [review]
No compilation errors ✅
Correct # of inversions ✅

**Pull request merged**